endpoints.json

```diff
@@ -60,5 +60,24 @@
         }
       ]
     }
+  },
+  "POST /api/articles/:article_id/comments":{
+    "description":"add a comment for an article (seleted by article ID), accept properties: username and body. Will responds with the posted comment.",
+    "queries": [],
+    "acceptProperties":{
```

Comment:
i'd change this to exampleRequest rather than acceptProperties to follow convention